config/routes.rb

```diff
@@ -1,5 +1,14 @@
 Rails.application.routes.draw do
-  get 'welcome/index'
 
+  resources :sessions, only: [:new, :create, :destroy]
```

Comment:
@katiekeel @yoasyo25 It appears the sessions routes are repeated. I think we should delete line 3.

Reply:
Yep 👍